docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,190 @@
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+.. contents:: 
+   :local:
+   :backlinks: none
+   :depth: 2
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+You will be controlling your Flocker cluster via the CLI you've installed locally.
+The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :width: 60 %
```

Comment:
Adam, I think you have my personal gmail address in your mail account and keep using it by mistake

> On 11 Jun 2015, at 09:02, Adam Dangoor <notifications@github.com> wrote:
> 
> In docs/using/tutorial/movingvolume.rst <https://github.com/ClusterHQ/flocker/pull/1506#discussion_r32197588>:
> 
> > +.. contents:: 
> > +   :local:
> > +   :backlinks: none
> > +   :depth: 2
> > +
> > +You will use Flocker to migrate a Docker container with its data volume from one host to another.
> > +The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
> > +
> > +To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
> > +You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
> > +
> > +You will be controlling your Flocker cluster via the CLI you've installed locally.
> > +The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI:
> > +
> > +.. image:: images/flocker-tutorial-initial-setup.svg
> > +   :width: 60 %
> Get rid of :width: and maybe it's too big for your liking some places, but it doesn't get too small.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ClusterHQ/flocker/pull/1506/files#r32197588>.
>